metrics/core/labels.go

```diff
@@ -135,6 +135,9 @@ var podLabels = []LabelDescriptor{
 
 var metricLabels = []LabelDescriptor{
 	LabelResourceID,
+	LabelAcceleratorMake,
```

Comment:
Adding these labels here adds these labels to a lot of metrics where they don't make sense at all. You should use `acceleratorLabels` instead.

Reply:
All the diff from this file can go away.

Reply:
Will do.